packages/expo-modules-core/ExpoModulesCore.podspec

```diff
@@ -34,29 +48,22 @@ Pod::Spec.new do |s|
   s.static_framework = true
   s.header_dir     = 'ExpoModulesCore'
 
-  header_search_paths = [
-  ]
-
   # Swift/Objective-C compatibility
   s.pod_target_xcconfig = {
     'USE_HEADERMAP' => 'YES',
     'DEFINES_MODULE' => 'YES',
     'CLANG_CXX_LANGUAGE_STANDARD' => 'c++20',
     'SWIFT_COMPILATION_MODE' => 'wholemodule',
-    'HEADER_SEARCH_PATHS' => header_search_paths.join(' '),
     'OTHER_SWIFT_FLAGS' => "$(inherited) #{fabric_enabled ? fabric_compiler_flags : ''}"
   }
-  user_header_search_paths = [
-    '"${PODS_CONFIGURATION_BUILD_DIR}/ExpoModulesCore/Swift Compatibility Header"',
-    '"$(PODS_ROOT)/Headers/Private/Yoga"', # Expo.h -> ExpoModulesCore-umbrella.h -> Fabric ViewProps.h -> Private Yoga headers
-  ]
   s.user_target_xcconfig = {
-    "HEADER_SEARCH_PATHS" => user_header_search_paths,
+    "HEADER_SEARCH_PATHS" => [
+      '"${PODS_CONFIGURATION_BUILD_DIR}/ExpoModulesCore/Swift Compatibility Header"',
+      '"$(PODS_ROOT)/Headers/Private/Yoga"', # Expo.h -> ExpoModulesCore-umbrella.h -> Fabric ViewProps.h -> Private Yoga headers
```

Comment:
```suggestion
```
maybe we don't need Yoga because react-native core will add Yoga for modules

Reply:
Sadly, it fails on compiling `AppDelegate.mm` – it can't find `<yoga/style/Style.h>` 🤔 I'll investigate this later.